app/src/main/res/layout/main.xml

```diff
@@ -99,7 +99,7 @@
               android:textStyle="bold" />
 
           </LinearLayout>
-
+          
```

Comment:
Please don't make formatting changes to irrelevant files

Reply:
@mhutti1 I have made the changes. Please review. I had made the changes because I had kept the seekbar here while texting.